bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -104,4 +104,12 @@ private FormConstants() {
     public static final String RT_FD_FORM_RESET_BUTTON_V1 = RT_FD_FORM_PREFIX + "actions/reset/v1/reset";
 
     public static final String FORM_FIELD_TYPE = "form";
+    public static final String RT_FD_FORM_RENDER_CONDITION_CUSTOM_PROPERTIES = RT_FD_FORM_PREFIX
```

Comment:
Please move this resource type to cq-guides, this logic is internal and I don't think customers should ever change this